examples/storm-kafka-client-examples/pom.xml

```diff
@@ -42,7 +42,9 @@
             <groupId>org.apache.storm</groupId>
             <artifactId>storm-kafka</artifactId>
             <version>${project.version}</version>
+            <!-- You can reduce jar size by uncommenting this and putting dependencies in $STORM-HOME/extlib instead of including them in the jar
```

Comment:
Same above. Maybe we can let users choose how they provide dependencies, change the sentence to `...uncommenting this and providing dependencies manually.` or so. My intention is that we don't recommend putting dependencies to extlib directory, unless they know what they're doing (affecting whole topologies' dependencies)